src/deployment.h

```diff
@@ -92,14 +94,20 @@ inline bool createAppRunHook(appdir::AppDir &appDir) {
         return false;
     }
 
+    const bool haveGtk3 = bf::exists(themesPath / "libqgtk3.so");
```

Comment:
Can we add a comment above this variable explaining that it's an either/or, i.e., there's either libqgtk3 or libqgtk2, please?